.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Test aubrey-transcription
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Just a thought: should we pin this down to a specific ubuntu release? I haven't mentioned it on other PRs, but the more I see it the more I wonder if at some point soon the latest ubuntu will soon stop providing unsupported Python versions like 3.6, at which point our tests would just stop working for that version. It's just a thought though, it's not that I think anything is _wrong_ with how it is.